lambda_functions/README.md

```diff
@@ -0,0 +1,41 @@
+# Healthcheck Lambdas
+
+The lambdas for the SML Portal healthcheck can be changed to do this you will need to follow some steps after making your code changes.
```

Comment:
Not sure what you mean by this statement, can you update it to be more explicit in your meaning.